docs/wrappers.md

```diff
@@ -1,14 +1,78 @@
-# Wrappers
+Gym includes numerous wrappers for environments that include preprocessing and various other functions. The following categories are included:
 
-## Space Wrappers
-Wrappers that transform observation and/or action space. Contains
- * Discretize (make a discrete version of a continuous space)
- * Flatten (put all actions/observations into a single dimension)
- * Rescale (rescale the range of values for continuous spaces).
+## Observation Wrappers
 
-Learn more here: https://github.com/ngc92/space-wrappers
+`DelayObservation(env, delay)` [text]
+* Bring over from SuperSuit
 
-## Utility wrappers for Atari Games
-The baseline repository contains wrappers that are used when doing Atari 
-experiments. 
-These can be found here: https://github.com/openai/baselines/blob/master/baselines/common/atari_wrappers.py
+`FilterObservation(env, filter_keys)` [text]
+* Needs review (including for good assertion messages and test coverage)
+
+`FlattenObservation(env)` [text]
+* Needs good asseration messages
+
+`FrameStack(env, num_stack, lz4_compress=False)` [text]
+* Needs review (including for good assertion messages and test coverage)
+
+`GrayScaleObservation(env)` [text] 
+* Needs R/G/B channel argument added like supersuit wrapper
+* Needs review (including for good assertion messages and test coverage)
+* Needs CV2 dependency replaced with SuperSuit's way of doing full grey scaling: https://github.com/PettingZoo-Team/SuperSuit/blob/master/supersuit/utils/basic_transforms/color_reduction.py
+
+`PixelObservationWrapper(pixels_only=True, render_kwargs=None, pixel_keys=("pixels",))` [text]
+* Needs review (including for good assertion messages and test coverage)
+
+`RescaleObservation(env, min_obs, max_obs`) [text]
+* Bring over from Supersuit or from https://github.com/openai/gym/pull/1635
+
+`ResizeObservation(env, shape)` [text]
+* Needs review (including for good assertion messages and test coverage)
+* Switch from CV2 to Lycon2 once released
+
+`TimeAwareObservation(env)` [text]
+* Needs review (including for good assertion messages and test coverage)
+
+## Action Wrappers
+
+`ClipAction(env)` [text]
+* Needs review (including for good assertion messages and test coverage)
+
+`ChangeDtype(env, dtype)` [text]
+* Bring over from SuperSuit
+
+`RescaleAction(env, min_action, max_action)` [text]
+* Needs review (including for good assertion messages and test coverage)
+
+`StickyActions(env, repeat_action_probability)` [text]
+* Create as separate wrapper from Atari/bring over from SuperSuit
+
+## Reward Wrappers
+
+`ClipReward(env, lower_bound-1, upper_bound=1)` [text]
+* Bring over from SuperSuit
+
+## Lambda Wrappers
+
+`ActionLambda(env, change_action_fn, change_space_fn)` [text]
+* Bring over from SuperSuit
+
+`ObservationLambda(env, observation_fn, observation_space_fn)` [text]
+* Bring over from SuperSuit, replaces TransformObservation
+
+`RewardLambda(env, reward_fn)` [text]
+* Bring over from SuperSuit, replaces TransformReward
```

Comment:
Any reason why renaming `TransformObservation` and `TransformReward`? This seems like a easily avoidable breaking change.